internal/testprotos/desc_test3.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto2";
```

Comment:
Instead of creating a new file in `testprotos`, let's just add stuff to the `test.proto` that lives in the `protoparse` package. It already imports `descriptor.proto` so you could just append something like this to it:


```proto
message MethodOpt {
	optional bool authed = 1;

	enum Action {
		LOGIN = 0;
	 	READ = 1;
	 	WRITE = 2;
	 }
	message Permission {
		optional Action action = 1;
		optional string entity = 2;
	}

 	repeated Permission permission = 2;
}

service TestService {
	rpc UserAuth(Test) returns (Test) {
		option (validator) = {
			authenticated: true
			permission: {
				action: LOGIN
				entity: "client"
			}
		};
	}
	rpc Get(Test) returns (Test) {
		option (validator) = {
			authenticated: true
			permission: {
				action: READ
				entity: "user"
			}
		};
	}
}
```

Reply:
Done.